### ISSUE TITLE
Microsoft.AspNetCore.Authentication.Negotiate fails at runtime with programs that use System.DirectoryServices.Protocols > 6.0.1

### ISSUE THREAD

**Issue by mvarblow** [dotnet/runtime#84031]

**Comment by laodc** [comment]
I think this also occurred for System.Management

**Comment by ViktorHofer** [comment]
> I think this also occurred for System.Management

Correct. Both `System.DirectoryServices.Protocols` and `System.Management` were impacted and new packages will be released for both 6.0.x and 7.0x with the next servicing release in May.

See https://github.com/dotnet/runtime/pull/84357 and https://github.com/dotnet/runtime/pull/84355 for more details.
